### PR TITLE
frontend: add resource name to SectionHeader

### PR DESCRIPTION
**Pull request by farodin91**
image

### CODE REVIEW

File: frontend/src/components/common/Resource/MainInfoSection/MainInfoSectionHeader.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ export function MainInfoHeader<T extends KubeObject>(props: MainInfoHeaderProps<
 
   return (
     <SectionHeader
-      title={title || (resource ? resource.kind : '')}
+      title={title || (resource ? `${resource.kind} (${resource.getName()})` : '')}
```
Comment (joaquimrocha) [inline frontend/src/components/common/Resource/MainInfoSection/MainInfoSectionHeader.tsx:121]: I think we can make it more readable by using `KIND: NAME` than the `()`, WDYT? Also saves 1 character 😉 
Example:
Screenshot with example

**Review by joaquimrocha — CHANGES_REQUESTED** [review]
Thanks! It makes sense but I left a suggestion on making the title read a bit more official and easier.

File: frontend/src/components/common/Resource/MainInfoSection/MainInfoSectionHeader.tsx (hunk 1 above)
Comment (farodin91) [reply frontend/src/components/common/Resource/MainInfoSection/MainInfoSectionHeader.tsx]: Looks great. I changed it to this way.

**Comment by farodin91** [comment]
@joaquimrocha I had to fix tests.

**Review by joaquimrocha — APPROVED** [review]

**Comment by joaquimrocha** [comment]
@farodin91 , thank you for the changes. This is a nice UX improvement.

**Pull request merged**